packages/reactivity/src/baseHandlers.ts

```diff
@@ -77,7 +80,15 @@ function createGetter(isReadonly = false, shallow = false) {
       return isReadonly
     } else if (
       key === ReactiveFlags.RAW &&
-      receiver === (isReadonly ? readonlyMap : reactiveMap).get(target)
+      receiver ===
+        (isReadonly
+          ? shallow
+            ? shallowReadonlyMap
+            : readonlyMap
+          : shallow
```

Comment:
Here mixed other unrelated code.

Reply:
Ah damn. Somehow some changes meant for another branch slipped in here.

Wil fix it in a minute